examples/graphql-context/graphql_context.bal

```diff
@@ -3,18 +3,17 @@ import ballerina/http;
 import ballerina/lang.value;
 
 @graphql:ServiceConfig {
-    // Initialization of the `graphqlContext` should be provided to the
-    // `contextInit` field.
+    // Initialization of the `graphqlContext` should be provided to the `contextInit` field.
     contextInit: isolated function (http:RequestContext requestContext,
                                     http:Request request)
                                     returns graphql:Context|error {
 
         // Initialize the `graphql:Context` object.
         graphql:Context context = new;
 
-        // Retrieve the header named `scope` and set it to the context with the
-        // `scope` key. If the header does not exist, this will return an
-        // `error`, and thereby, the request will not be processed.
+        // Retrieve the header named `scope` and set it to the context with the `scope` key. If the
+        // header does not exist, this will return an `error`, and thereby, the request will not be
+        // processed.
```

Comment:
We should not add these extra line breaks in the comments, right?

Reply:
Yes, we need not add line breaks up to the character limit of 100.

Reply:
set the length of the comments to 100 chars.